test/extended/images/mongodb_replica.go

```diff
@@ -0,0 +1,114 @@
+package images
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+)
+
+var _ = g.Describe("[images][mongodb] openshift mongodb replication", func() {
+	defer g.GinkgoRecover()
+
+	const (
+		templatePath         = "https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json"
+		deploymentConfigName = "mongodb"
+		expectedValue        = `{ "status" : "passed" }`
+		insertCmd            = "db.bar.save(" + expectedValue + ")"
+	)
+
+	const (
+		expectedReplicasAfterDeployment = 3
```

Comment:
this test doesn't run for any PR on the mongodb repo.  so basically we'd find out after the mongodb PR merged, when this test starts breaking.  but i think we can live with that.  If we can actually get to the point where this test runs regularly and people notice when it breaks, i'd be thrilled :)